.rubocop.yml

```diff
@@ -103,35 +88,14 @@ Style/ExpandPathArguments:
 Style/GlobalStdStream:
   Enabled: false
 
-Style/HashEachMethods:
-  Enabled: false
-
 Style/HashLikeCase:
   MinBranchesCount: 4
 
-Style/HashTransformKeys:
-  Enabled: false
```

Comment:
TIL I learned about transform_keys, but again since it can generate false positives I prefer no to enable it by default

Reply:
https://docs.rubocop.org/rubocop/usage/auto_correct.html#safe-auto-correct

maybe we don't need to worry too much about the unsafe cops since we can use a safe autocorrect?

Reply:
We can also `AutoCorrect: false` so it never happens and it has to be a manual choice